server.js

```diff
@@ -89,6 +136,10 @@ const check_timew = ({on_exit}) => {
     })
 
     rl.on('close', (line) => {
-	if (!response_sent) on_exit(false)
+        if (!response_sent) on_exit(false)
     })
 }
+
+const spawn_tw = (opts) => {
```

Comment:
It was about time to make a function for this :D Thanks for not being a pig like the original author of this code! ;-)